rclcpp/include/rclcpp/node_interfaces/node_base_interface.hpp

```diff
@@ -112,6 +112,21 @@ class NodeBaseInterface
     rclcpp::CallbackGroupType group_type,
     bool automatically_add_to_executor_with_node = true) = 0;
 
+  RCLCPP_PUBLIC
```

Comment:
Add some comments on new interface.  
e.g.  

/// Return a callback group which is executed in internal executor.